js/app/src/lite/css.ts

```diff
@@ -35,10 +35,11 @@ export async function wasm_proxied_mount_css(
 	});
 	const css = new TextDecoder().decode(response.body);
 
-	const existing_link = document.querySelector(
+	// Gradio Lite can be reloaded without refreshing the page, so we need to remove the existing style element if it exists.
+	const existing_style = document.querySelector(
 		`style[data-wasm-path='${url_string}']`
 	);
-	if (existing_link) return;
```

Comment:
This was implemented after the non-Wasm ver. which skips mounting the CSS if there is always a mounted element as below.
https://github.com/gradio-app/gradio/blob/28342a204001c98b86628635582fcf54aa0855aa/js/app/src/css.ts#L17-L19

However, Lite is different from the normal one in that the app can be executed repeatedly and the frontend can be refreshed many times, so this design was not correct and the requested CSS must be mounted, replacing the old one.